core/metrics/src/service.rs

```diff
@@ -0,0 +1,38 @@
+use actix_web::{web, Responder, Scope};
+use futures::lock::Mutex;
+use lazy_static::lazy_static;
+use std::sync::Arc;
+
+use ya_service_api_interfaces::Provider;
+
+use crate::metrics::Metrics;
+
+pub struct MetricsService;
+
+lazy_static! {
+    static ref METRICS: Arc<Mutex<Metrics>> = Metrics::new();
+}
```

Comment:
There's no need for lazy static as long we don't use both gsb and rest.

Reply:
it was failing when was not static since we have multiple workers / threads serving REST API